File: iotcloud-restapi-java/Core/src/com/qcloud/restapi/iotcore/response/GetCreateMultiDevTaskResponse.java
package com.qcloud.restapi.iotcore.response;

import com.google.gson.*;

public class GetCreateMultiDevTaskResponse extends Response {
    /**
     * 解析JSON字串并构造GetCreateMultiDevTaskResponse对象
     *
     * @param response 从服务端返回的应答JSON字串
     */
    public GetCreateMultiDevTaskResponse(String response) {
        super(response);
    }

    public int getTaskStatus() {
        try {
            if (jsonObj != null) {
                return jsonObj.get("taskStatus").getAsInt();
            }
        }catch (Exception e) {
        }

        return -1;
    }

    public String getTaskID() {
        try {
            if (jsonObj != null) {
                return jsonObj.get("taskID").getAsString();
            }
        }catch (Exception e) {
        }

        return "";
    }
}

File: iotcloud-restapi-php/Core/src/IoTCloud/Response/GetCreateMultiDevTaskResponse.php
<?php

namespace TXIoTCloud\Response;

require_once "CreateMultiDeviceResponse.php";

class GetCreateMultiDevTaskResponse extends CreateMultiDeviceResponse
{

    /**
     * 任务是否完成。0 代表任务未开始，1 代表任务正在执行，2 代表任务已完成
     */
    private $taskStatus;

    /**
     * CreateMultiDeviceResponse constructor.
     */
    public function __construct()
    {
    }

    /**
     * 获取任务是否完成。0 代表任务未开始，1 代表任务正在执行，2 代表任务已完成
     */
    public function getTaskStatus()
    {
        return $this->taskStatus;
    }

    /**
     * 设置任务是否完成。0 代表任务未开始，1 代表任务正在执行，2 代表任务已完成
     * @param $taskStatus
     */
    public function setTaskStatus($taskStatus)
    {
        $this->taskStatus = $taskStatus;
    }

}